### DISCUSSION TITLE
Weitere Funktionen der Wallbox

### DISCUSSION THREAD

**Discussion by ollip995** [skipperro/enpal-homeassistant#5] [Ideas]
Hi. 

Erstmal danke für deinen Code. 
Ist es geplant zukünftig weitere Funktionen der Wallbox mit einzubinden? 
Ich denke da an den „Laden starten“ und „Laden stoppen“ Button sowie einen Schalter mit dem man zwischen dem „Eco“ und „Schnell“ Lademodus umschalten kann. 

Grüße, Olli.

**Comment by skipperro** [comment]
Geplant - ja.
Aber wann - keine Ahnung. 

Enpal hat für neue Kunde eine neue System gebaut, der mit meine Installation nicht kompatibel ist und deswegen kann ich keine neue Funktionen entwickeln oder testen. 

Mehr dazu ist hier: https://github.com/Skipperro/enpal-homeassistant/issues/1

**Comment by skipperro** [comment]
@ollip995 Ich habe Update von Enpal bekommen und https://github.com/Skipperro/enpal-homeassistant/issues/1 geschlossen.
Ich werde in kommenden Tagen schauen, ob es möglich ist der Wallbox via Home Assistant zu steuern.

**Comment by skipperro** [comment]
@ollip995 Leider hat die Home Assistant Integration nur Leserechte für die InfluxDB von Enpal, sodass ich nur die verfügbaren Daten lesen, aber nichts ändern kann. 
Das bedeutet, dass meine Hände gebunden sind, es sei denn, Enpal stellt API-Endpunkte zur Steuerung der Wallbox zur Verfügung. 
Ich werde weiterhin danach fragen, aber ich glaube nicht, dass diese Funktion in absehbarer Zeit verfügbar sein wird.

**Comment by ollip995** [comment]
Schade. Danke für dein Update. Am 17.05.2024 um 09:19 schrieb Tomasz Dylewski ***@***.***>:﻿
@ollip995 Leider hat die Home Assistant Integration nur Leserechte für die InfluxDB von Enpal, sodass ich nur die verfügbaren Daten lesen, aber nichts ändern kann.
Das bedeutet, dass meine Hände gebunden sind, es sei denn, Enpal stellt API-Endpunkte zur Steuerung der Wallbox zur Verfügung.
Ich werde weiterhin danach fragen, aber ich glaube nicht, dass diese Funktion in absehbarer Zeit verfügbar sein wird.
